cmd/ar+https/main.go

```diff
@@ -18,11 +18,16 @@ import (
 	"bufio"
 	"context"
 	"os"
+	"fmt"
 
 	"github.com/GoogleCloudPlatform/artifact-registry-apt-transport/apt"
 )
 
 func main() {
 	apt := apt.NewAptMethod(bufio.NewReader(os.Stdin), os.Stdout)
-	apt.Run(context.Background())
+	err := apt.Run(context.Background())
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "%+v\n", err)
+		os.Exit(100)
```

Comment:
Q: is there a reason to return `100` specifically, instead of the more typical `1` or `-1`?

Reply:
Nice, sounds good to me. Let's return 100 on error cases.